src/applications/personalization/view-representative/components/ViewRepresentativeDetails.jsx

```diff
@@ -15,6 +15,12 @@ export const ViewRepresentativeDetails = props => (
         <strong>Phone: </strong>
         {props.telephone}
       </p>
+      {props?.searchRepresentative && (
+        // add valid href when search rep url has been determined
+        <a role="button" target="_self" className="usa-button" href="#">
```

Comment:
Could you get away with the `manifest.json`'s `rootUrl`? One less thing to remember down the road!

Reply:
We don't have a search app spun up just yet, but I've updated this to `/view-change-representative/search`, since it seems plausible that could be the url for the app in the future.